aiida-registry-app/src/Components/MainIndex.jsx

```diff
@@ -235,6 +249,42 @@ export function MainIndex() {
           </Box>
           </div>
 
+          {isSearchSubmitted === true ? (
+            <>
+            <h2>Showing {searchResults.length} pages matching the search query.</h2>
+            {searchResults.length === 0 && (
+              <div>
+            <h3 className='submenu-entry' style={{textAlign:'center'}}>Can't find what you're looking for?<br/> Join the AiiDA community on Discourse and open a discussion here. <br/> <a href='https://aiida.discourse.group' target="_blank">https://aiida.discourse.group</a></h3>
+            </div>
+            )}
```

Comment:
Fantastic! Just a minor request here. There seems a blank area in the box:
![image](https://github.com/aiidateam/aiida-registry/assets/8831685/fa16bc16-bed5-47dd-acbd-1e6a1df10419)

Can you try to fix it?